cluster/mutex.go

```diff
@@ -106,12 +106,12 @@ func (m *Mutex) refreshLock() error {
 // Lock locks m. If the mutex is already locked by any plugin instance, including the current one,
 // the calling goroutine blocks until the mutex can be locked.
 func (m *Mutex) Lock() {
-	m.LockWithContext(context.Background())
+	_ = m.LockWithContext(context.Background())
 }
 
-// LockWithContext locks m unless the context is cancelled. If the mutex is already locked by any plugin
+// LockWithContext locks m unless the context is canceled. If the mutex is already locked by any plugin
```

Comment:
Ah well, the Canadian in me tried :)

Reply:
Personally I would love to move it back, but go's linting is aggressively American. :)